src/Ryujinx/Input/GTK3/GTK3KeyboardDriver.cs

```diff
@@ -6,12 +6,12 @@
 
 namespace Ryujinx.Input.GTK3
 {
-    public class GTK3KeyboardDriver : IGamepadDriver
+    public class Gtk3KeyboardDriver : IGamepadDriver
```

Comment:
```suggestion
    public class GTK3KeyboardDriver : IGamepadDriver
```
Same as other comment